### ISSUE TITLE
Move buttons don't do anything 

### ISSUE THREAD

**Issue by Jontux** [makerbase-mks/MKS-H43#8]
When I home the printer it works well but when I try to move any plane with x+, x-, y+, y-, z+ or z- button, it doesn't do anything.

I'm using MKS H43 v1.0 with v3.1 firmware. 
Mainboard is SKR V1.4 Turbo with TMC2209. 
Marlin firmware is 2.0-MKS-H43.

I don't know if this is because of Marlin firmware config or a bug in MKS H43 firmware.

**Comment by Voxel3DPrinting** [comment]
Homing works you say? and what if you try connecting it to a PC with pronterface?

**Comment by Jontux** [comment]
Yes homing works well and with pronterface I can move all axels, but moving with a H43 doesn't work.
